client/src/api/getProduct.js

```diff
@@ -0,0 +1,16 @@
+import axiosConfig from '../axiosConfig';
+
+const getProduct = async (tourId) => {
+  try {
+    const { data, status } = await axiosConfig(`/products/${tourId}`);
+
+    if (status === 200) {
+      return data;
+    }
+  } catch (err) {
```

Comment:
<img width="807" alt="Снимок экрана 2022-12-13 в 16 55 43" src="https://user-images.githubusercontent.com/107925398/207367032-e85f6777-77a4-4b34-a9b4-a4cfaebaca84.png">

eslint waiting for return something in other cases then status === 200

Reply:
the rule is already disabled since we dont need to always explicitly return a value from our async functions.

its a dev server bug,  just stop ur dev server and re run again with npm start and the rule will take effect.